components/feature/addons/src/main/res/layout/mozac_feature_addons_fragment_dialog_addon_installed.xml

```diff
@@ -84,6 +85,8 @@
         android:layout_marginTop="16dp"
         android:layout_marginEnd="16dp"
         android:layout_marginBottom="16dp"
+        android:paddingStart="20dp"
```

Comment:
I think the padding here is 6 as per https://github.com/mozilla-mobile/fenix/issues/11023#issuecomment-640347274

Reply:
The comment says that the "Okay, Got it" button should match the padding of the "Allow" button which is 20dp. Unless I'm reading it wrong?